core/frontend/src/components/vehiclesetup/params/subParams.ts

```diff
@@ -0,0 +1,90 @@
+import { SemVer } from 'semver'
+
+import { Dictionary } from '@/types/common'
+
+import { heavySitlParams, standardSitlParams } from './subSitlParams'
+
+const POWER_SENSE_MODULE = {
+  BATT_CAPACITY: 0.0,
+  BATT_AMP_OFFSET: 0.330,
+  BATT_AMP_PERVLT: 37.8788,
+  BATT_VOLT_MULT: 11.000,
+  BATT_MONITOR: 4,
+}
+
+const SERVOS_STANDARD_PIXHAWK = {
+  SERVO7_FUNCTION: 59, // RCIN9 (camera)
+  SERVO8_FUNCTION: 7, // mount_tilt
+  SERVO8_REVERSED: 1,
+}
+
+const SERVOS_HEAVY = {
+  SERVO7_FUNCTION: 39, // Motor7
+  SERVO8_FUNCTION: 40, // Motor8
+  SERVO8_REVERSED: 0,
+  SERVO9_FUNCTION: 59, // RCIN9 (camera)
+  SERVO10_FUNCTION: 7, // mount_tilt
+  SERVO10_REVERSED: 1,
+}
+
+const PIXHAWK1 = {
+  LEAK1_PIN: 55,
+}
+
+const NAVIGATOR = {
+  LEAK1_PIN: 27,
+}
+
+const BASE_BLUEROV = {
+  ...POWER_SENSE_MODULE,
+  AHRS_ORIENTATION: 16,
+  BRD_RTC_TYPES: 3,
+  MNT_RC_IN_TILT: 8,
+  MNT_TYPE: 1,
+  SERIAL0_PROTOCOL: 2, // MAVLink2
+}
+
+const BLUEROV_STANDARD = {
+  ...BASE_BLUEROV,
+  ATC_ANG_RLL_P: 0.0,
+  FRAME_CONFIG: 1,
+}
+
+const BLUEROV2_HEAVY = {
+  ...BASE_BLUEROV,
+  ATC_ANG_PIT_P: 6.0,
+  ATC_ANG_RLL_P: 6.0,
+  ATC_ANG_YAW_P: 6.0,
+  FRAME_CONFIG: 2,
+}
+
+function getSubParamSets(board: string | undefined, version: SemVer | undefined): Dictionary<Dictionary<number>> {
```

Comment:
what is the user case for undefined board ?

Reply:
no use case case, it is just happens before the board type is loaded from the backend

Reply:
So the function should not accept this case.